hydromt/raster.py

```diff
@@ -2340,28 +2353,61 @@ def to_slippy_tiles(
         }.get(driver.lower(), {})
         kwargs = {**kwargs0, **kwargs}
 
+        # Setting up information for zoomlevel calculation and
+        # determination of tile windows
+        # This section is purely for the resolution
```

Comment:
Could this section be moved inside the if `max_lvl is None:` statement?

Reply:
Done.